common/src/types/v0/store/switchover.rs

```diff
@@ -5,26 +5,22 @@ use crate::types::v0::{
     },
     transport::VolumeId,
 };
-
 use chrono::{DateTime, Utc};
 use serde::{Deserialize, Serialize};
+use std::net::SocketAddr;
 
 /// Defines operation for SwitchOverSpec.
 #[derive(Serialize, Deserialize, Debug, Clone)]
 pub enum Operation {
-    /// Initialize SwitchOverSpec.
+    /// Initialize switchover request.
     Init,
-    /// Shutdown original/old volume target of SwitchOverSpec.
-    ShutdownOriginal,
-    /// Create new volume target for SwitchOverSpec.
-    ReconstructTarget,
-    /// Switch volume target to newly created target.
-    SwitchOverTarget,
-    /// Delete old volume target.
+    /// Shutdown original/old volume target. Create new nexus for existing vol obj.
+    RepublishVolume,
+    /// Delete original/old volume target
     DeleteTarget,
-    /// Publish updated path of volume.
+    /// Publish updated path of volume to node-agent.
```

Comment:
Can we reorder Publish with Delete here?

Reply:
Incorporated

Reply:
Incorporated